.gitpod.yml

```
@@ -0,0 +1,31 @@
+tasks:
+  - name: Stargate
+    command: docker-compose up stargate
+  - name: api
+    command: |
+      sleep 35
+      docker-compose up eddiehub-api
+  - name: Eddiebot
+    init: |
+      npm i -g npm
+      npm install
+      gp await-port 3000
+    command: npm run start:dev
+
+github:
+  prebuilds:
+    # enable for the default branch (defaults to true)
+    master: true
```

We use `main` here, not `master`.
```suggestion
    main: true
```